### PR TITLE
Allow both &topic and &category as place page url param

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: server/routes/place.py [hunk 1]
```diff
@@ -47,14 +47,17 @@
 @bp.route('/<path:place_dcid>', strict_slashes=False)
 def place(place_dcid=None):
     dcid = flask.request.args.get('dcid', None)
-    topic = flask.request.args.get('topic', None)
+    category = flask.request.args.get('category', None)
+    if not category:
+        # Used to use topic as the url param.
+        category = flask.request.args.get('topic', None)
```
Comment (beets) [inline server/routes/place.py:53]: I think we should also issue a redirect, if not from search, from &topic to &category so we don't increase the number of url's in the search index (and for consistency)

**Review by beets — COMMENTED** [review]
can we also update the homepage url's? thx!

File: server/routes/place.py (hunk 1 above)
Comment (shifucun) [reply server/routes/place.py]: Always redirect to ?category now.

File: server/tests/place_test.py [hunk 2]
```diff
@@ -96,6 +101,11 @@ def test_place(self, mock_get_place_type, mock_get_i18n_name):
         assert response.status_code == 200
```
Comment (beets) [inline server/tests/place_test.py:101]: does this need follow_redirects=True? (likewise for the other test that still uses the topic param)

**Review by beets — APPROVED** [review]
thanks for adding the webdriver!

File: server/tests/place_test.py (hunk 2 above)
Comment (shifucun) [reply server/tests/place_test.py:101]: Done

**Pull request merged**